### DISCUSSION TITLE
`DEFAULT` value in update queries

### DISCUSSION THREAD

**Discussion by wanjas** [prisma/prisma#8435] [Q&A] [answered]
Hello!

Is it possible to use DEFAULT in update queries without resorting to the raw sql? 

I mean an equivalet to the following: `UPDATE "Table" set "fieldName" = DEFAULT WHERE ...`

**Comment by ryands17** [comment, marked as answer]
@wanjas 👋 

This isn't possible currently. It would be great if you could create a feature request for this [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) so that we can look into this.

**Reply by wanjas** [reply]
Thank you! I will

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
